src/PersonalSite.Tests/PersonalSite.Domain.API.UnitTests/PersonalSite.WebApi.UnitTests.csproj

```diff
@@ -9,6 +9,9 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="Moq" Version="4.16.1" />
```

Comment:
I've been planning to use Moq instead NSubstitute for the entire project soon, so I would prefer to do it everything at once so we close that part and remove NSubstitute package reference, we can do it together next week